test/jdk/sun/net/www/http/KeepAliveCache/KeepAliveProperty.java

```diff
@@ -171,6 +171,7 @@ public static void main(String args[]) throws Exception {
             if (!expectClose)
                 throw e;
         }
+        s.join();
```

Comment:
LGTM. The logger at line 141 should be stashed into a static field too to make sure it will not be GC'ed before being used.